pkg/pull/pull.go

```diff
@@ -277,7 +277,8 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 		IsGKEAutopilot:      k8sutil.IsGKEAutopilot(clientset),
 		IncludeMinio:        pullOptions.IncludeMinio,
 	}
-	if err := upstream.WriteUpstream(u, writeUpstreamOptions); err != nil {
+	installationManifest, err := upstream.WriteUpstream(u, writeUpstreamOptions)
```

Comment:
the `Installation` manifest gets updated later in the function when processing and writing the midstream. the `WriteUpstream` shouldn't return a manifest here, we should just run `kotsKinds.Marshal(....)` for the installation crd at the end before writing the kots kinds.